db/migrate/20211123060535_create_treatment_inertia_tables.rb

```diff
@@ -0,0 +1,35 @@
+class CreateTreatmentInertiaTables < ActiveRecord::Migration[5.2]
+  def up
+    create_table :clean_medicine_to_dosage, id: false do |t|
```

Comment:
Just `clean_medicines`/ `clean_medications`?

Reply:
I'll reconsider all names after the first pass. I've mostly gone with the names that were used in the spike.

Reply:
Had a chance to reconsider this with https://github.com/simpledotorg/simple-server/pull/3091. I went with `clean_medicine_to_dosages` since we are also cleaning up dosages here and converting them to floats, and it's easy to miss that if we name it as `clean_medicines`.

Reply:
`clean_medicine_dosages`